### DISCUSSION TITLE
挂载rootfs只读怎么办

### DISCUSSION THREAD

**Discussion by wyxdlz54188** [PenUniverse/PenMods#263] [你问我答]
![Screenshot_20241103_155925_com termux](https://github.com/user-attachments/assets/c84f4a71-e29d-4be2-baff-7eb5b0a2e510)

**Comment by wyxdlz54188** [comment]
![Screenshot_20241103_155925_com termux](https://github.com/user-attachments/assets/63e5a99e-e821-4362-b087-779a8d8eb1ed)

**Comment by wyxdlz54188** [comment]
对了sed -i命令替换后我在termux中试了一下报错

**Comment by SkySight-666** [comment]
mount -w?

**Comment by wyxdlz54188** [comment]
哥删不了我发你源包吧



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月4日(周一) 晚上10:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 挂载rootfs只读怎么办 (Discussion #263)




 
mount -w?

**Reply by SkySight-666** [reply]
> 哥删不了我发你源包吧
> […](#)
> ---原始邮件--- 发件人: ***@***.***&gt; 发送时间: 2024年11月4日(周一) 晚上10:33 收件人: ***@***.***&gt;; 抄送: ***@***.******@***.***&gt;; 主题: Re: [PenUniverse/PenMods-release] 挂载rootfs只读怎么办 (Discussion #263) mount -w? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

bushi，我在binwalk这个固件不开 -Me 没分析出来真正的rootfs
然后，既让主控是rk，回读可行否？

**Reply by SkySight-666** [reply]
> > 哥删不了我发你源包吧
> > […](#)
> > ---原始邮件--- 发件人: _**@**_._**> 发送时间: 2024年11月4日(周一) 晚上10:33 收件人: _**@**_.**_>; 抄送: _**@**_._**_**@**_.**_>; 主题: Re: [PenUniverse/PenMods-release] 挂载rootfs只读怎么办 (Discussion #263) mount -w? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: _**@**_.***>
> 
> bushi，我在binwalk这个固件不开 -Me 没分析出来真正的rootfs 然后，既让主控是rk，回读可行否？

开了Me我虚拟机空间直接炸了  60G 不够它装的 分析出来一堆垃圾

**Comment by wyxdlz54188** [comment]
我用DNA解压出rootfs了但是无法合成img



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月4日(周一) 晚上11:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] 挂载rootfs只读怎么办 (Discussion #263)




  
哥删不了我发你源包吧
 …
 ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年11月4日(周一) 晚上10:33 收件人: @.&gt;; 抄送: @.@.&gt;; 主题: Re: [PenUniverse/PenMods-release] 挂载rootfs只读怎么办 (Discussion #263) mount -w? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***&gt;
  
bushi，我在binwalk这个固件不开 -Me 没分析出来真正的rootfs
 然后，既让主控是rk，回读可行否？
